### ISSUE TITLE
Periods in translation tables

### ISSUE THREAD

**Issue by innerpeace0** [angular-translate/angular-translate#1115]
Which is better:

`"SENTENCE": "This is a complete sentence. "`
`<span translate>SENTENCE</span>`

or

`"SENTENCE": "This is a complete sentence"`
`<span><span translate>SENTENCE</span>. </span>`

**Comment by tspaeth** [comment]
We really love to help and support, but these kind of "how to" questions really fit better on http://stackoverflow.com - please ask there as you'll reach a bigger user base audience.

This is the main issue/defect/feature tracker for the module and only in rare cases a basic first level support forum. Thanks.

**Comment by innerpeace0** [comment]
I will post there, but, even though it's a larger audience pool, I'm not sure it has a larger pool of people who know angular-translate. These kind of newbie questions could be classified as issue/defect/feature trackers for your docs and faq to help them become more complete.

**Comment by innerpeace0** [comment]
I just signed up at stackoverflow.com and, as I suspected, the angular-translate pool is quite small. Of all the questions tagged angular-translate, about half have no answers at all and only 18 have been answered in the past month. While stackoverflow might be a place to _ask_ newbie questions about angular-translate, I'm not sure that it will result in many useful _answers,_ which is what I (and, I expect, other newbies) really want. Thanks.
